### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Matthieu96Code**
### In this pull request the following tasks were implemented:

-  Create a method called `current_user` in ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user`.
    - Create a comment on behalf of the `current_user`.
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -9,5 +9,26 @@ def show
...
+  end
+
+  def create
+    @post = current_user.posts.new(post_parameters)
+    if @post.save
+      redirect_to user_post_path(current_user, @post)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/posts_controller.rb]: - [x] I could not create a post, look at this short [video](https://www.loom.com/share/b92018d3ade840f0af3b24035d5e4d89?sid=b780c367-def4-4cfc-9fc6-6ef97e014580) to understand what I mean

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Matthieu96Code  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Matthieu96Code** [comment]
Hi @VICTOR-CHUKWU,
Thank you for reviewing my project.
I tried again on my end, and everything is working perfectly.

blog app forms

but I'm still checking what is wrong

**Comment by Matthieu96Code** [comment]
Hi @VICTOR-CHUKWU,
Thank you for reviewing my project.
I'm solving the issues.

**Review by TadesseDev — APPROVED** [review]
### Hi @Matthieu96Code 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

- N/A


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**